tests/empty-comment-test.hs

```diff
@@ -0,0 +1,74 @@
+-- |
```

Comment:
This file doesn't actually seem to be used in a test suite?

Reply:
Sorry I didn't see this earlier. I've got the hang of hlint `.test` testing now and converted that one `.hs` file into a `.test` file with multiple tests within it.